ci/server.sh

```diff
@@ -1,11 +1,14 @@
-# build the server
-mvn clean package -DskipTests
+#!/bin/bash
+set -ev
 
-# start the server and capture the pid
-java -jar target/ground-0.1-SNAPSHOT.jar server conf/config.yml &
+# Build the server.
+sbt clean package
+
+# Start the server and capture the pid.
+java -jar target/scala-2.11/ground_2.11-0.1-SNAPSHOT.jar server conf/application.yml &
```

Comment:
@vsreekanti There's still something wrong with this command but I'm not too familiar with how to start the server under Play Framework; any ideas?